### PR TITLE
Completed javascript-this exercise.

### PR DESCRIPTION
**Pull request by sallysujilee**
Screenshot 2023-04-17 at 3 18 58 PM

### CODE REVIEW

File: javascript-this/javascript-this-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# javascript-this-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is `this` in JavaScript?
+it's an implicit parameter
```
Comment (robgmerrill) [inline javascript-this/javascript-this-quiz-notes.md:8]: Take another shot at this after Q&A

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! One quiz question answer is worth revisiting :)

File: javascript-this/javascript-this-quiz-notes.md (hunk 1 above)
Comment (sallysujilee) [reply javascript-this/javascript-this-quiz-notes.md:8]: I wrote more but I moved it to the following question
i'll fix it

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**